files/sysbus/webos-connman-adapter.perm.json

```diff
@@ -2,22 +2,41 @@
     "com.palm.wifi": [
         "settings",
         "devices",
-        "networking.internal", 
-        "public"
+        "networking.internal"
+    ],
+    "com.webos.service.wifi": [
+        "settings",
+        "devices",
+        "networking.internal"
     ],
     "com.palm.connectionmanager": [
         "settings",
         "devices",
         "networking.internal"
     ],
+    "com.webos.service.connectionmanager": [
+        "settings",
+        "devices",
+        "networking.internal"
+    ],
     "com.palm.wan": [
         "settings",
         "devices",
         "networking.internal"
     ],
+    "com.webos.service.wan": [
+        "settings",
+        "devices",
+        "networking.internal"
+    ],
     "com.palm.pan": [
         "settings",
         "devices",
         "networking.internal"
+    ],
+    "com.webos.service.pan": [
```

Comment:
where does this ".pan"/".wan" service name come from ? I don't see it in the rest of your PR, is it a name that the adapter is register to too ?

Reply:
Also, maybe webos-connman-adapter.role.json.in needs some modification too, because there are only two allowedNames there